Sources/DNA/Color/Color+Button.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import Foundation
+
+public extension UIColor {
```

Comment:
I think I made a mistake when I made this extension public. I don't think its actually necessary since its an extension. What do you think?
https://stackoverflow.com/questions/33925840/access-control-for-swift-extensions

Reply:
I think so too, the extension should inherit the access level of UIColor so making it `public` is redundant.

Reply:
Nice!